### PR TITLE
Scanamo 0.9.2

### PR DESCRIPTION
**Pull request by philwills**
This finally allows us to get rid of the bit horrible comment!

### CODE REVIEW

File: app/data/Recipes.scala [hunk 1]
```diff
@@ -39,25 +41,18 @@ object Recipes {
...
-      baseImage = baseImage,
-      roles = roles,
-      modifiedBy = modifiedBy,
-      modifiedAt = DateTime.now(),
-      bakeSchedule = bakeSchedule
+    bakeSchedule: Option[BakeSchedule])(implicit dynamo: Dynamo): Either[DynamoReadError, Recipe] = {
+
+    val update = table.update('id -> recipe.id,
+      set('description -> description) and
+        set('baseImageId -> baseImage.id) and
```
Comment (philwills) [inline app/data/Recipes.scala]: This extra indent puts my teeth on edge, but I haven't dived into scalariform to work out how to fix it.

File: app/data/Recipes.scala [hunk 2]
```diff
@@ -39,25 +41,18 @@ object Recipes {
...
+        set('roles -> roles) and
+        set('modifiedBy -> modifiedBy) and
+        set('modifiedAt -> DateTime.now()) and
+        (if (bakeSchedule.isDefined) set('bakeSchedule -> bakeSchedule) else remove('bakeSchedule))
     )
-    // TODO This is a bit horrible. We have to get the record from Dynamo just to copy the next build number over. Really we want to do a partial update.
-    // This is currently blocked on guardian/scanamo#62.
```
Comment (thomaska) [inline app/data/Recipes.scala]: The fact that this is issue is still open is not accidental I assume right ?!

Comment (philwills) [inline app/data/Recipes.scala]: I was planning on closing it, once this is merged.

Comment (cb372) [inline app/data/Recipes.scala]: I demand proof!

Comment (philwills) [inline app/data/Recipes.scala]: https://www.theguardian.com/stage/2013/mar/24/proof-menier-chocolate-factory-review

**Pull request merged**